schema/core/primitive/graph.json

```diff
@@ -14,6 +14,9 @@
             },
             {
                 "$ref": "graph/named_node_in_group.json"
+            },
+            {
+                "$ref": "graph/typed_node.json"
```

Comment:
This name seems a bit weird - typed = typed on a typewriter. If we really need it, we can call it node_with_type?
Also `type` is overloaded in this context.